lib/airbrake-ruby/config/validator.rb

```diff
@@ -0,0 +1,69 @@
+module Airbrake
+  class Config
+    ##
+    # Validates values of {Airbrake::Config} options.
+    #
+    # @api private
+    # @since v1.5.0
+    class Validator
+      ##
+      # @return [String]
+      REQUIRED_ID_KEY_MSG = 'both :project_id and :project_key are required'.freeze
+
+      ##
+      # @return [String]
+      WRONG_ENV_TYPE_MSG = "the 'environment' option must be configured " \
+                           "with a Symbol (or String), but '%s' was provided: " \
+                           "%s".freeze
+
+      ##
+      # @return [Array<Class>] the list of allowed types to configure the
+      #   environment option
+      VALID_ENV_TYPES = [NilClass, String, Symbol].freeze
+
+      ##
+      # @return [String] error message, if validator was able to find any errors
+      #   in the config
+      attr_reader :error_message
+
+      ##
+      # Validates given config and stores error message, if any errors were
+      # found.
+      #
+      # @param config [Airbrake::Config] the config to validate
+      def initialize(config)
+        @config = config
+        @error_message = nil
+      end
+
+      ##
+      # @return [Boolean]
```

Comment:
Not sure why you think so. It will return Boolean and the tests prove this:
https://github.com/airbrake/airbrake-ruby/pull/115/files/14231445e58bcae105865ff5512b68c9e51e4c91#diff-c7a89698d88722526dd499d062dca276R6